masq/tests/chain_specific_directory_test_integration.rs

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) 2019, MASQ (https://masq.ai) and/or its affiliates. All rights reserved.
+
+use crate::utils::{DaemonProcess, MasqProcess};
+use masq_lib::utils::find_free_port;
+use std::thread;
+use std::time::Duration;
+
+mod utils;
+
+#[test]
+fn ensure_data_directory_has_specific_chain_direcotry_within() {
+    let port = find_free_port();
+    let daemon_handle = DaemonProcess::new().start(port.clone());
+    thread::sleep(Duration::from_millis(3000));
```

Comment:
This `sleep` is unnecessary. By the time `start()` finishes, the Daemon is already up and ready for instructions via `port`.